README.md

```diff
@@ -1 +1,68 @@
-# opa-java-client
\ No newline at end of file
+# opa-java-client
+Opa java client is a wrapper for Opa rest API. The goal was to create client that is lightweight and framework independent. It's built for current Bisnode needs, including:
+ - creating documents
+ - creating policies
+ - querying for documents
+## Installation
+**Prerequisites:** Java 11 or higher
+
+Add library using maven:
+```xml
+<dependency>
+    <groupId>com.bisnode.opa</groupId>
+    <artifactId>opa-java-client</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+</dependency>
+```
+or Gradle
+```groovy
+implementation 'com.bisnode.opa:opa-java-client:0.1.0-SNAPSHOT'
+```
+
+## Usage
+
+###Query for document
```

Comment:
I think you'll need to add a whitespace after the three hashbangs or they won't be parsed as H3s.